lang/src/lib.rs

```diff
@@ -710,3 +723,39 @@ macro_rules! source {
         }
     };
 }
+
+#[test]
+fn test_instruction_data() {
```

Comment:
Could you move this to [`lang/tests`](https://github.com/coral-xyz/anchor/tree/4c0af6dc5394c714a70296eae686b261f520f206/lang/tests)?

Reply:
will do -- some time in the next 0-2 days.